### ISSUE TITLE
Additional references for italic and bold

### ISSUE THREAD

**Issue by jsnmrs** [double-great/listen#116]
Reference to add to https://doublegreat.dev/listen/bold-and-italics/:
- [em and strong are not announced by screen readers](https://www.stefanjudis.com/today-i-learned/em-and-strong-are-not-be-announced-by-screen-readers/)
